rootfs/root/deboot/root/bin/install.sh

```diff
@@ -4,8 +4,8 @@
 dpkg-divert --local --rename --add /sbin/initctl
 ln -sf /bin/true /sbin/initctl
 
-# localedef -i de_DE -c -f UTF-8 de_DE.UTF-8 
-echo "Europe/Berlin" >/etc/timezone
+timedatectl set-timezone Europe/Berlin
+ln -sf /usr/share/zoneinfo/Europe/Berlin /etc/localtime
 dpkg-reconfigure --frontend noninteractive tzdata
```

Comment:
Warum wird die Einstellung der Zeitzone eigentlich vor dem update/upgrade gemacht?  
Wird damit sicher gestellt, dass Datum und Zeit auf der Maschine stimmen, und ist das Voraussetzung für korrekte Funktion vom update/upgrade?

Reply:
There is no reason. The time settings have to be set anyway. The update/upgrade has to be done for any other additional installations. Both processes are completely independent.